### PR TITLE
doc: update release v0.1 release notes

### PR DESCRIPTION
**Pull request by dbkinder**
Include developer comments, add commit change log

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

File: doc/release_notes.rst [hunk 1]
```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
 * The ACRN Block, NIC, and console Virtio drivers
```
Comment (gvancuts) [inline doc/release_notes.rst]: These three lines (all three about virtio) really feel weird to me. Are these really 3 SW components? I would have called out the hypervisor, device model, tools and acrn-kernel instead. Perhaps even highlighting GVT-g in this release there.

File: doc/release_notes.rst [hunk 2]
```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
...
+  - GVT-g for ACRN (a.k.a ACRNGT): GPU sharing feature enabled. Service OS
+    and User OS can run GPU workload simultaneously.
+  - Direct display supported by ACRNGT. Service OS and User OS are each
+    assigned to different display. The display ports supports eDP, HDMI.
+  - Surface sharing supported by ACRNGT. SOS can access an individual
+    surface or a set of surfaces from UOS without having to access the
+    entire frame buffer of UOS.
+  - See :ref:`APL_GVT-G-hld` documentation for more information.
+
+* Virtio network supported. SOS and UOS can share physical LAN network
```
Comment (gvancuts) [inline doc/release_notes.rst]: How about rephrasing this to "Virtio standard supported". At the moment, the first sentence talks about the network but the rest of the paragraph talks about both network and device storage (which really is what virtio is about, i.e. both network and device storage).

File: doc/release_notes.rst [hunk 3]
```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
...
-* Sometimes system hangs, especially when workload is high (e.g. running benchmarks, playing videos)
+Black screen displayed after booting SOS/UOS
+  The ``weston`` display server, window manager, and compositor used by ACRN
+  (from Clear Linux) may not have been properly installed and started.
+  **Workaround** is described in ACRN GitHub issue :acrn-issue:`663`.
+
+SSD Disk ID not consistent between SOS/UOS
+  The SSD disk ID in the UOS is not the same as in the SOS when
+  the SSD device is pass-through to the UOS. The ID is also changing after
+  a reboot.  This shouldn't' be the case.  **Workaround:** None.
```
Comment (gvancuts) [inline doc/release_notes.rst]: You have an extra apostrophe at the end of "shouldn't"

File: doc/release_notes.rst [hunk 4]
```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
...
+  There is no impact to functionality. The issue will be fixed in the
+  next release.
+
+Hypervisor and DM version numbers incorrect
+  The command ``/usr/bin/acrn-dm --version`` does not show the correct
+  build number.  **Workaround":** None.  The issue will be fixed in the
```
Comment (gvancuts) [inline doc/release_notes.rst]: Extra quote (") character after "Workaround"

File: doc/release_notes.rst [hunk 5]
```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
...
+  next release.
+
+
+Change Log
+==========
+
+These commits have been added to the acrn-hypervisor repo since our
+initial release in March 2018 (click on the CommitID link to see details):
```
Comment (gvancuts) [inline doc/release_notes.rst]: You say "March 2018" here but the comment below indicates "--after="2018-05-11""?

File: doc/conf.py [hunk 6]
```diff
@@ -32,7 +32,13 @@
 # ones.
 
 sys.path.insert(0, os.path.join(os.path.abspath('.'), 'extensions'))
-extensions = ['breathe', 'sphinx.ext.graphviz', 'kerneldoc']
+extensions = ['breathe', 'sphinx.ext.graphviz', 'sphinx.ext.extlinks', 'kerneldoc']
+
+# extlinks provides a macro template
+
+extlinks = {'acrn-commit': ('https://github.com/projectacrn/acrn-hypervisor/commit/%s', ''),
```
Comment (gvancuts) [inline doc/conf.py]: Very cool mechanism these `acrn-commit` and `acrn-issue`.. I like them!! :-)

We should also add them to the doc guidelines.

File: doc/release_notes.rst (hunk 1 above)
Comment (dbkinder) [reply doc/release_notes.rst]: That was from the previous release notes.  I'm just going to delete that section.

File: doc/release_notes.rst (hunk 5 above)
Comment (dbkinder) [reply doc/release_notes.rst]: That's because there's no git history before Fri May 11 14:44:28 2018.  I'll just delete mentioning March 2018 here.

**Comment by dbkinder** [comment]
Thanks @gvancuts We'll see if we get more comments from the China team as they wake up this morning :)

**Comment by gvancuts** [comment]
> That's because there's no git history before Fri May 11 14:44:28 2018. I'll just delete mentioning March 2018 here.

Interestingly, I used the exact command-line to get the list but it seemed to royally ignore the date and gave me all commits from the very start of the repo (5th of March).

**Comment by gvancuts** [comment]
> :acrn-commit:`981dd9a` Initial commit Fri May 11 14:44:27 2018 +0800

That's true (I did not have the %cd in my command-line). This must be an artefact of the fiddling that happened when we merged the three repos into one. If you do a 'git show 981dd9a', you will see it's actually from the 5th of March.

**Comment by dbkinder** [comment]
Yep.  The **author date** notes when this commit was originally made (i.e. when you finished the git commit). The **commit date** gets changed every time the commit is modified.

    git log  --pretty=fuller 981dd9a
    commit 981dd9ac889be9f3723471188e9f97741750f280
    Author:     Rraghuveer <35912739+Rraghuveer@users.noreply.github.com>
    AuthorDate: Mon Mar 5 13:51:03 2018 -0800
    Commit:     lijinxia <jinxiax.li@intel.com>
    CommitDate: Fri May 11 14:44:27 2018 +0800

        Initial commit

Looks like ``--before`` and ``--after`` refer to the CommitDate

**Pull request merged**